src/registry_schemas/example_data/schema_data.py

```diff
@@ -2601,6 +2601,30 @@
     'type': 'CLIENT'
 }
 
+CORRECTION_CONVERSION = {
+    'filing': {
+        'header': {
+            'name': 'correction',
+            'certifiedBy': 'full name',
+            'email': 'no_one@never.get',
+            'date': '2020-02-18',
+            'waiveFees': True
+        },
+        'business': {
+            'foundingDate': '2007-04-08T00:00:00+00:00',
+            'identifier': 'FM1104477',
+            'legalName': 'legal name - FM1104477',
+            'legalType': 'SP'
+        },
```

Comment:
I believe we are moving away from including the `header` and `business` block in the example data.

Instead, follow how `CORRECTION_CP_SPECIAL_RESOLUTION` has things structured for the example data and the corresponding tests.  A recent conversation about can be found [here](https://github.com/bcgov/business-schemas/pull/138#discussion_r1231301574).

Reply:
I fixed